uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChange.php

```diff
@@ -343,6 +341,23 @@ public static function getTotalBalanceChangeForLeaveRequest(LeaveRequest $leaveR
   }
 
   /**
+   * Returns the LeaveBalanceChange for the TOIL request ID
```

Comment:
The method description is not accurate. It says it returns a LeaveBalanceChange, but it actually returns the amount of a LeaveBalanceChange